views/bigquery.go

```diff
@@ -203,6 +203,10 @@ func buildViewsEventsQuery(table string, spec QuerySpec) (string, []interface{},
 		if !ok {
 			return "", nil, fmt.Errorf("invalid breakdown field: %s", by)
 		}
+		// skip breakdowns that are already in the query
+		if strings.Contains(query.Columns().ToSql().Sql, field) {
```

Comment:
WDYT of?
```suggestion
		if field == "playback_id" && spec.Filter.PlaybackID != "" {
		  // A playback ID filter already groups by the field as well, so skip it here
```

Reply:
The underlying problem was that when playback Id is passed as a filter then it's added to the query twice as a column and a group by. 

I think this would mess up the case when playback Id is in the breakdown by 

We just need to skip it if it's already there. I don't know how to access the query's existing columns and group by's

Reply:
oh! on second look it only ignores playbackid if is in the filter right? could we have the same problem with dstorageurl?